ambari-funtest/src/test/resources/stacks/HDP/2.1.1/upgrades/upgrade_to_new_stack.xml

```diff
@@ -187,13 +191,17 @@
             <command>ls</command>
           </task>
         </pre-upgrade>
+        <pre-downgrade/>
       </component>
       <component name="NODEMANAGER">
         <pre-upgrade>
           <task xsi:type="execute">
             <command>ls</command>
           </task>
         </pre-upgrade>
+        <pre-downgrade/>
+        <post-downgrade/>
+        <post-upgrade/>
```

Comment:
Looks these are duplicates.Please handle.

Reply:
corrected it.